create-workspace.yml

```diff
@@ -3,8 +3,8 @@ platform: linux
 image_resource:
   type: docker-image
   source:
-    repository: ((dataworks.terraform_repository))
-    tag: ((dataworks.terraform_version))
+    repository: hashicorp/terraform
+    tag: 0.12.31
```

Comment:
```suggestion
    repository: ((dataworks.terraform_repository))
    tag: ((dataworks.terraform_12_version))
```
Does this not work?

Reply:
Fixed this.

Reply:
It was when I tested a repo with that version and the docker image wasn't present